js/main-pin.js

```diff
@@ -6,7 +6,24 @@
     RADIUS: 32
   };
 
+  var PinDragArea = {
+    MIN_X: 0,
+    MAX_X: 1200,
+    MIN_Y: 130,
+    MAX_Y: 630,
+  };
+
   var mainPin = document.querySelector('.map__pin--main');
+
+  var defaultMainPinCoords = {
+    x: mainPin.offsetLeft,
+    y: mainPin.offsetTop,
+  };
+  var resetMainPinPosition = function () {
```

Comment:
См. [замечание из прошлого ревью](https://github.com/htmlacademy-javascript/987137-keksobooking-19/pull/4#discussion_r387365038).

Подобная небрежность может стать причиной отказа для собеседования.
```suggestion
  };
  
  var resetMainPinPosition = function () {
```

Reply:
поняла, приняла!